app/Navbar.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Image from 'next/image';
+
+export default function Navbar() {
+	return (
+		<nav className="h-14 w-screen bg-white shadow-lg shadow-gray-400 sm:h-20 lg:mb-12 lg:h-24">
```

Comment:
![image](https://github.com/ezerssss/unihub-web/assets/72904036/fb5c5721-865f-48b5-80f2-5a352898dcad)

theres a horizontal scrollbar, seems like the navbar is overflowing the screen width wise


you can remove the `w-screen` it should still work